ansible/library/minigraph_facts.py

```diff
@@ -349,7 +349,7 @@ def parse_xml(filename, hostname):
             hwsku = child.text
 
     # port_alias_map maps ngs port name to sonic port name
-    if hwsku == "Force10-S6000":
+    if hwsku == "Force10-S6000" or hwsku == "ACS-MSN2700":
```

Comment:
I notice that switch2.xml file is also describing ACS-MSN2700. So it is better to replace the current minigraph interfaces' name to Ethernet*. So we don't need to have such logic here.